test/integration/features/cmd-config.feature

```diff
@@ -213,7 +213,7 @@ user defined options which changes default behaviour of Minishift.
 
   Scenario: Checking that disk-size value was applied
      When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/sda: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
+     Then stdout should match "Disk \/dev\/(s|v)da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
```

Comment:
@agajdosi Can't we use `?da` as regex where we don't care what is the first char? `?da` is what we are using as part of disk check for preflight checks.

Reply:
Yes `?da` is better choice +1

Reply:
@praveenkumar @budhrg Good point, missed that. Updating now.